jstp/src/main/java/com/metarhia/jstp/connection/Connection.java

```diff
@@ -638,14 +763,29 @@ public void setClientMethodNames(String interfaceName, List<String> names) {
     clientMethodNames.put(interfaceName, methods);
   }
 
+  /**
+   * Checks if socket is connected
+   *
+   * @return is socket connected
```

Comment:
Same as in socket description 'true if connection is established and false otherwise'